documentation/asciidoc/topics/proc_securing_custom_configuration.adoc

```diff
@@ -34,8 +34,16 @@ include::yaml/credential_store.yaml[]
 ----
 
 . Apply the changes.
-. Define a `credential-reference` in your configuration with `store="credentials"`.
+. Open your {brandname} Server for editing.
+. Add a `credential-reference` in the `security`  section of the configuration.
```

Comment:
This isn't correct. A `credential-reference` can appear in many parts of the configuration, for example in a DataSource connection factory or a Security-realm.

https://infinispan.org/docs/stable/titles/server/server.html#credential-keystore-references_credential-keystores

I think we can skip the `security` section part of this bullet and just say something like "Add a `credential-reference` to your configuration".